src/applications/mhv/secure-messaging/components/MessageList/FolderHeader.jsx

```diff
@@ -38,6 +40,16 @@ const FolderHeader = props => {
     );
   };
 
+  useEffect(
+    () => {
+      if (folder?.folderId > 0) {
+        focusElement(document.querySelector('h1'));
+        document.title = `${folder.name} ${PageTitles.PAGE_TITLE_TAG}`;
+      }
+    },
+    [folder],
+  );

```

Comment:
I am guessing it is working fine as is, but perhaps would make a better sense to incorporate on a parent level like `FolderThreadListView.jsx` as it appears to be more relevant to entire container rather than just his header component. Just a suggestion, not necessarily it is the best way to go. Leaving up to you to make that call.